cmd/calyptia/delete_core_instance.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/calyptia/api/types"
+	"github.com/calyptia/cli/k8s"
+	"github.com/spf13/cobra"
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func newCmdDeleteCoreInstance(config *config, testClientSet kubernetes.Interface) *cobra.Command {
+	var aggregatorName string
+	var skipError bool
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+	configOverrides := &clientcmd.ConfigOverrides{}
+	cmd := &cobra.Command{
+		Use:   "instance",
+		Short: "Delete a core instance",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			if aggregatorName == "" {
+				return fmt.Errorf("the core instance name must be provided")
+			}
+			agg, err := config.cloud.Aggregators(ctx, config.projectID, types.AggregatorsParams{
+				Name: &aggregatorName,
```

Comment:
On other commands we can find by either ID or name.
That a look at how is done in other commands.

The arg can be autocompleted too.